app.py

```diff
@@ -21,10 +20,17 @@ def index():
     queue_count = get_queued_count()
     recently_placed_count = get_recently_placed_count()
     recently_processed_count = get_recently_processed_count()
+    scenarios = [
+        { 'display': 'Add broken order', 'value': 'BrokenOrder' },
```

Comment:
I'm assuming this was added after deploying - it doesn't show up on [the example I can see](http://test-group-test-user1-order-processing-app-service.azurewebsites.net/) but happy to believe it's just a later change